src/utilities.js

```diff
@@ -107,13 +107,34 @@ export function getResults(searchType, obj) {
     },
   };
 
+  // Create array of items to query
+  var comparisonItems = [];
```

Comment:
use `let` and change the name to `comparisonItem` so that way we can use shorthand notation below:
`let comparisonItem;`